File: 1781-check-if-two-string-arrays-are-equivalent/1781-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {

        StringBuilder res1 = new StringBuilder();
        StringBuilder res2 = new StringBuilder();

        for(String i: word1){
            res1.append(i);
        }

        for(String i: word2){
            res2.append(i);
        }

        if(res1.toString().equals(res2.toString())){
            return true;
        }
        return false;
        
    }
}

File: 1781-check-if-two-string-arrays-are-equivalent/1781-check-if-two-string-arrays-are-equivalent.py
class Solution:
    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:

        res1 =''
        res2=''

        for i in word1:
            res1+=i
        for i in word2:
            res2+=i

        if (res1==res2):
            return True
        
        return False

        